### ISSUE TITLE
Debug "auditing failed" errors

### ISSUE THREAD

**Issue by razvan-moj** [ministryofjustice/cloud-platform#3493]
narrowing down from https://github.com/ministryofjustice/cloud-platform/issues/3429, this is to debug a specific error, as we have many of them: for a query like
```
fields @timestamp, @message
| filter @message like "Auditing failed of  request"
| sort @timestamp desc
```
we get 12k events per day:

<img width="931" alt="Screenshot 2022-01-24 at 12 07 59" src="https://user-images.githubusercontent.com/75040720/150780327-12493f37-1b34-411d-b9e0-d3254a36ed0f.png">

**Comment by razvan-moj** [comment]
https://console.aws.amazon.com/support/home?region=eu-west-2#/case/?displayId=9668306651&language=en

**Comment by razvan-moj** [comment]
From https://us-east-1.console.aws.amazon.com/support/home?region=eu-west-2#/case/?displayId=9842546541&language=en :
The problem that you are facing is an upstream issue that was fixed in version 1.22 of Kubernetes in this commit [1].
Therefore, the best option is to wait for release 1.22 of EKS and upgrade your control plane to fix the issue.

Will revisit when we're on 1.22
